sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -412,8 +412,27 @@ public static class SubStatusCodes {
 
         // Client generated request rate too large exception
         public static final int THROUGHPUT_CONTROL_BULK_REQUEST_RATE_TOO_LARGE = 10005;
-
+      
         public static final int USER_REQUEST_RATE_TOO_LARGE = 3200;
+
+        //SDK Codes(Client)
+        public static final int TRANSPORT_GENERATED_410 = 20001;
+        public static final int TIMEOUT_GENERATED_410 = 20002;
+        public static final int TRANSPORT_GENERATED_503 = 20003;
+        public static final int CPU_OVERLOAD = 20004;
+        public static final int THREAD_STARVATION = 20005;
```

Comment:
I do not think we check THREAD_STARVATION in java

Reply:
please remove `CPU_OVERLOAD`, `THREAD_STARVATION` for now as they are not being used as of today.